.pipelines/azure-pipeline-config-tests.yml

```diff
@@ -831,19 +1128,24 @@ stages:
           DisableRemediation: false
 
   - deployment: Deploy_AKS_Chart
-    displayName: "Deploy: AKS dev cluster"
-    environment: Prometheus-Collector
+    displayName: "Deploy: AKS tests cluster"
```

Comment:
Thanks Rashmi, I am thinking that instead of getting a new build each night in this pipeline, the main build pipeline could also deploy to this cluster for merges to main. Then this cluster will have the latest changes and each night the rest of this pipeline can still run the tests on that cluster

Reply:
Yes. Thats a good point @gracewehner. I can clean the build steps from this pipeline and that way we dont have to maintain 2 copies of the pipeline.